### DISCUSSION TITLE
Seperate database for module

### DISCUSSION THREAD

**Discussion by mortezashoghi** [oqtane/oqtane.framework#1942] [Q&A]
We want to create oqtane modules for get data from another databases which has different structures .
Is it possible ?
what is your solution?

**Comment by leigh-pointer** [comment]
Hi @mortezashoghi check out this thread [#1836](https://github.com/oqtane/oqtane.framework/discussions/1836) this should give you a good insight.

**Reply by mortezashoghi** [reply]
Thanks.
Problem solved.
